.eslintrc

```diff
@@ -11,5 +11,26 @@
   "rules": {
     "no-console": 1, // Means warning
     "prettier/prettier": 2 // Means error
-  }
+  },
+  "overrides": [
+    {
+      "files": ["**/*.ts"],
+      "env": { "browser": true, "es6": true, "node": true },
+      "extends": [
+        "eslint:recommended",
+        "prettier"
+      ],
+      "parser": "@typescript-eslint/parser",
+      "parserOptions": {
+        "ecmaVersion": 2018,
+        "sourceType": "module",
+        "project": "./tsconfig.json"
+      },
+      "plugins": ["@typescript-eslint","prettier"],
+      "rules": {
+        "no-console": 1, // Means warning
```

Comment:
In the rules for TypeScript files, you'll want to add:

```
                "no-undef": "off",
                "no-unused-vars": "off"
```

TypeScript itself will catch these in compilation, and it does it a lot better than eslint does.  Otherwise once you start importing types (unless you're 100% sure to always use Typescript 4's `import type` declaration), you'll get unused import errors in TS4.

Reply:
sorry I just realised, that this one was not resolved.